config/schema/og.schema.yml

```diff
@@ -110,6 +110,9 @@ og.og_role.*:
     group_bundle:
       type: string
       label: 'Group bundle'
+    is_admin:
```

Comment:
@amitaibu OH, so OgAccess::userAccess collects all permissions, which would be specific to a node bundle for example? I thought users can just have an admin role for e.g. 1 group and not another?